docs/usage/self-hosted-configuration.md

```diff
@@ -1093,6 +1110,18 @@ The combinations of `requireConfig` and `onboarding` are:
 | `requireConfig=optional` | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, the repository will be processed. | Repository is processed regardless of config file presence.   |
 | `requireConfig=ignored`  | No onboarding PR will be created and repo will be processed while ignoring any config file present.                                                     | Repository is processed, any config file is ignored.          |
 
+## s3Endpoint
+
+If set, Renovate will use this string as the `endpoint` when instantiating the AWS S3 client.
```

Comment:
Is there a simpler word than `instantiate` that we can use here?

Maybe `when starting the AWS S3 client`?